### BIG differences, this is diff of wrong files.
### CLUS "Type ATOM"
### and CLHS topic selection, http://clhs.lisp.se/Body/a_atom.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:types:atom.txt	2017-04-27 22:01:09.000000000 +0100
+++ saved-pages-clus/cl:types:atom.txt	2017-04-27 22:01:09.000000000 +0100
@@ -1,4 +1,13 @@
-ATOM
-Please select which reference to ATOM you intended
+Type ATOM

-FunctionTypeGlossary Entry
\ No newline at end of file
+Supertypes
+atom, t
+Description
+It is equivalent to (not cons).
\ No newline at end of file
